src/styles/stylesheet.scss

```
@@ -39,7 +39,7 @@ textarea.#{getGlobal("CONFIG.CLASS.codeInput")}, .#{getGlobal("CONFIG.CLASS.code
 
 ##{getGlobal("CONFIG.ID.preferencesShortcut")} {
     background-image: url("#{getGlobal("SITE.URL_ICONS_GENERAL")}") !important; // to override Blargmode
-    background-position: -164.5px -95px;
+    background-position: -165px -94px;
```

The height shouldn't be modified.

```suggestion
    background-position: -165px -95px;
```

---

It looks much more centered on `-94px`, but whatever works for you.